### ISSUE TITLE
external datagroup skipCertificateCheck

### ISSUE THREAD

**Issue by tizianorosato-gr** [F5Networks/f5-appsvcs-extension#511]
### Is your feature request related to a problem? Please describe.
Apparently I can't create an external datagroup from a file stored on git repo.
Looks like "skipCertificateCheck" option is not available for external datagroups

### Describe the solution you'd like
```json
{
    "class": "AS3",
    "action": "deploy",
    "persist": true,
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.29.0",
        "remark": "Another description",
        "T-TEST-DGext": {
            "class": "Tenant",
            "App1": {
                "class": "Application",
                "template": "generic",
                "DATAGROUP_TEST": {
                    "class": "Data_Group",
                    "storageType": "external",
                    "keyDataType": "string",
                    "remark": "JIRA TICKET..",
                    "separator": ":=",
                    "externalFilePath": "https://git.private.net/myrepo/datagroups/DATAGROUP_TEST.txt",
		    "skipCertificateCheck": true
                }
            }
        }
    }
}
```

### Describe alternatives you've considered
none

### Additional context
Using BigIP 14.1.x and AS 3.29.0

**Comment by dstokesf5** [comment]
Thank you for your feedback. This request has been added to our internal product backlog as AUTOTOOL-2876.

**Comment by tizianorosato-gr** [comment]
@dstokesf5 any chance to get this available with upcoming AS3 release?

**Comment by dstokesf5** [comment]
This should be available in the 3.34.0 release, which should be available by the end of January.

**Comment by dstokesf5** [comment]
This is resolved in the 3.34.0 release that is now available.
